eventsources/sources/awssqs/start.go

```diff
@@ -185,3 +192,44 @@ func fetchMessages(ctx context.Context, q *sqslib.SQS, url string, maxSize, wait
 	}
 	return result.Messages, nil
 }
+
+func (el *EventListener) createAWSSession(log *zap.SugaredLogger) (*session.Session, error) {
+	sqsEventSource := &el.SQSEventSource
+	awsSession, err := awscommon.CreateAWSSessionWithCredsInVolume(sqsEventSource.Region, sqsEventSource.RoleARN, sqsEventSource.AccessKey, sqsEventSource.SecretKey, sqsEventSource.SessionToken)
+	if err != nil {
+		log.Errorw("Error creating AWS credentials", zap.Error(err))
```

Comment:
Since error is being returned, no need to print it out here - there's already particular log message "failed to create aws session for" wrapped in the returned error to help debug.

Reply:
Done